src/openrct2/localisation/StringIds.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include "../common.h"
+#include "../sprites.h"
```

Comment:
Why is this needed?

Reply:
It's a leftover from when `EXPECTED_COUNT` was defined in the string array, now renamed and moved to the sprite enum. This include should be removed.

Reply:
Removed the redundant include